include/wpe/input.h

```diff
@@ -92,6 +92,9 @@ struct wpe_input_pointer_event {
 enum wpe_input_axis_event_type {
     wpe_input_axis_event_type_null,
     wpe_input_axis_event_type_motion,
+    wpe_input_axis_event_type_motion_smooth,
+
+    wpe_input_axis_event_type_2d = 1 << 16,
```

Comment:
Why using `1 << 16` for this value of the enum?

Reply:
It would serve as a mask value. If detected, the wpe_input_axis_event pointer would be casted to the wpe_input_axis_2d_event type and handled as such.